docs/src/pages/components/menus/ContextMenu.js

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect } from 'react';
+import Menu from '@material-ui/core/Menu';
+import MenuItem from '@material-ui/core/MenuItem';
+import Button from '@material-ui/core/Button';
+
+const ContextMenu = () => {
+  const [anchorEl, setAnchorEl] = React.useState(null);
+  const [mouseX, setMouseX] = React.useState(null);
+  const [mouseY, setMouseY] = React.useState(null);
+
+  const handleClick = event => {
+    event.preventDefault();
+    setMouseY(event.clientY - 4);
+    setMouseX(event.clientX + 2);
+    setAnchorEl(event.currentTarget);
+  };
+
+  useEffect(() => {}, [mouseX, mouseY]);
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  return (
+    <div
+      className="context-wrapper"
+      aria-controls="simple-menu"
+      aria-haspopup="true"
```

Comment:
Maybe @eps1lon has some insight on how to make this demo accessible.

Reply:
Would love to learn.